browser/mocha.log

```diff
@@ -0,0 +1,173 @@
+
+
+  Bounds parse() tests
+    Bounds.parse() call with an empty string argument
+incomplete rectangle
+      ✓ should return undefined
+    Bounds.parse() call with an string argument with 3 numbers
+incomplete rectangle
+      ✓ should return undefined
```

Comment:
Since this is unrelated to the fileserver, best to leave it out. If need help with removing this file from your commit, just ask. Happy to help with git commands.